.circleci/config.yml

```diff
@@ -191,8 +191,6 @@ jobs:
             cd packages/graphql-transformers-e2e-tests/ && yarn e2e
             --maxWorkers=3
           no_output_timeout: 90m
-          environment:
-            AMPLIFY_CLI_DISABLE_LOGGING: "true"
```

Comment:
Is this removal intentional?

Reply:
No I think it got auto generated

Reply:
Please revert this back. This was added recently as a part of this PR - https://github.com/aws-amplify/amplify-cli/pull/7087/files

Reply:
Found out this needs to get added into the base config as well
https://github.com/aws-amplify/amplify-cli/pull/7093

Reply:
cc: @SwaySway